### PR TITLE
Small fixes to bank-transfer language

### PR DESCRIPTION
**Pull request by lindeloev**

### CODE REVIEW

File: components/BankTransferStep.tsx [hunk 1]
```diff
@@ -32,9 +32,9 @@ export const BankTransferStep = () => {
       </table>
 
       <p>
-        Du kan opsætte månedlige overførsler i din netbank eller mobilbank med
-        den samme besked. Så sender vi donationen til det valgte velgørende
-        formål og indberetter evt. fradrag.
+        Du kan lavere en eller flere overførsler med ovenstående besked i din netbank eller mobilbank.
```
Comment (maximbaz) [inline components/BankTransferStep.tsx:35]: Typo in "Du kan lave"?

File: components/BankTransferStep.tsx [hunk 2]
```diff
@@ -32,9 +32,9 @@ export const BankTransferStep = () => {
...
+        Så sender vi donationen til det valgte velgørende formål og indberetter evt. fradrag.
+        Vi indrapporterer kun det, der faktisk er overført. Du vælger altså selv beløb og tidspunkt(er).
```
Comment (maximbaz) [inline components/BankTransferStep.tsx:37]: Maybe we shouldn't so "willingly" offer people to change amount from what they've just selected in the previous step? We sure _support_ that, but when amount changes, what basically happens is that I fill out the form for them with a new amount... 😅 On the other hand, this (almost) never happened yet, people do follow what they selected on the form, or fill it out again, so maybe I'm over thinking it...

**Review by maximbaz — COMMENTED** [review]
Thanks! 🥳

File: components/BankTransferStep.tsx (hunk 1 above)
Comment (lindeloev) [reply components/BankTransferStep.tsx:35]: Yes, good catch!

File: components/BankTransferStep.tsx (hunk 2 above)
Comment (lindeloev) [reply components/BankTransferStep.tsx:37]: Just wait till you see me donating ;-) 

OK, yes, I agree it's better to remove that last sentence "Du vælger altså selv beløb og tidspunkt(er)".

**Comment by maximbaz** [comment]
I applied the agreed changes and merged myself, thanks again 😉 https://github.com/giveffektivt/donation-platform/commit/2cddc3f064242254af932538b72d165de131d9b6

**Pull request closed without merging**